cdap-app-fabric-tests/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/WorkflowStatsSLAHttpHandlerTest.java

```diff
@@ -98,14 +97,17 @@ private void setStartAndRunning(ProgramId id, String pid, Map<String, String> ru
   @Test
   public void testStatistics() throws Exception {
     deploy(WorkflowApp.class, 200);
+    ApplicationDetail appDetails = getAppDetails(NamespaceId.DEFAULT.getNamespace(), "WorkflowApp");
```

Comment:
Please use constant for repeated string literal. It ensures there are no typos

Reply:
Or alternatively, create ApplicationId using ApplicationDetail data and not constants

Reply:
Fixed. now applicationId is created with appDetails